Sources/Contentful/Client.swift

```diff
@@ -298,27 +302,18 @@ open class Client {
             completion(Result.error(SDKError.unparseableJSON(data: data, errorMessage: "The SDK was unable to parse the JSON: \(error)")))
         }
     }
-    
-    deinit {
-        urlSession.invalidateAndCancel()
-    }
 }
 
 extension Client {
 
+
```

Comment:
Why the extra whitespace here?

Reply:
i have my linter settings such that, at most, 2 consecutive blank lines are allowed. So this is just a mistake not caught by the linter.